issue_15/script.js

```diff
@@ -0,0 +1,87 @@
+const showBtn = document.getElementById('js-show-btn');
+const modal = document.getElementById('js-modal');
+const closeBtn = document.getElementById('js-close-btn');
+const submitBtn = document.getElementById('js-submit-btn');
```

Comment:
nits
I think better not to omit variable names.
Share Mr.Moriken's comment.
https://github.com/seeeen044/JavaScript_handson/pull/19#discussion_r886921653

Reply:
https://github.com/yoshikawa-akr/js-lessons/pull/15#discussion_r990937928